posts/code-retreat.md

```diff
@@ -0,0 +1,9 @@
+Each year GISincers from all across the United States set aside their project work and get on a plane (or in a car) and travel to Unplugged, our annual company gathering.  Unplugged 2017 was held just a few weeks ago in Stone Mountain, GA and featured a new and exciting event for developers: Code Retreat.
```

Comment:
GISincers seems innocuous enough but you may want to check with marketing on how to refer to GISinc employees. They may have a specific branding in mind.